### PR TITLE
Add computeNextSnapshot function and testing.

### PR DESCRIPTION
**Pull request by ancamaria99**

### CODE REVIEW

File: step-release-vis/src/app/services/environment_test.ts [hunk 1]
```diff
@@ -15,7 +15,48 @@ describe('EnvironmentService', () => {
...
+
+    // @ts-ignore
+    const result: [TimestampUpperBoundSet, number] = service.computeNextSnapshot(inputCandInfo, inputSet);
+
+    expect(result[0].snapshot[0].position).toEqual(50);
+    expect(result[0].snapshot[1].position).toEqual(100);
+    expect(result[1]).toEqual(1);
+  });
+
+  it('#computeNextSnapshot candidateInfo[] empty', () => {
```
Comment (mracz) [inline step-release-vis/src/app/services/environment_test.ts:33]: we should not use variable names in the test description. Those can change pretty frequently. Lets say something like "Candidate list is empty" or "Candidate info list is empty"

File: step-release-vis/src/app/services/environment_test.ts [hunk 2]
```diff
@@ -15,7 +15,48 @@ describe('EnvironmentService', () => {
...
+    inputSet.orderMap.set('1', 0);
+    inputSet.snapshot[0] = new PolygonUpperBoundYPosition('1', 100);
+
+    // @ts-ignore
+    const result: [TimestampUpperBoundSet, number] = service.computeNextSnapshot(emptyCandInfo, inputSet);
+
+    expect(result[0]).toBe(inputSet);
+  });
+
+  it('#computeNextSnapshot with empty set', () => {
```
Comment (mracz) [inline step-release-vis/src/app/services/environment_test.ts:45]: based on the name of the test I can not distinguish between this and the above one, lets be more descriptive.

File: step-release-vis/src/app/services/environment_test.ts [hunk 3]
```diff
@@ -15,7 +15,48 @@ describe('EnvironmentService', () => {
   });
 
   // TODO(naoai): write getPolygons test
-  it('#getPercentages should return 3 candidates with ~ equal percentages', () => {
+
+  it('#computeNextSnapshot insert new candidate', () => {
```
Comment (mracz) [inline step-release-vis/src/app/services/environment_test.ts:19]: inserts

Comment (ancamaria99) [reply step-release-vis/src/app/services/environment_test.ts]: All done in new commits, thanks

**Pull request merged**